app/build.gradle

```diff
@@ -200,6 +200,25 @@ android {
         }
     }
 
+    def shouldUseStaticVersionCode = { ->
```

Comment:
please move this to the place where we have some other functions that read values from the user properties file